content/posts/fedora-eln.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Working on the next major RHEL release, in your upstream repo"
+date: 2020-10-04T12:07:03+02:00
+draft: false
+---
+
+[Fedora EL Niño](https://docs.fedoraproject.org/en-US/eln/) (ELN) is such an
+awesome idea. It enables building rawhide packages in two distinct buildroots:
+
+1. the standard Fedora Rawhide buildroot
+2. and the second one, which mimics Red Hat Enterprise Linux
+
+This way you can make sure that your new upstream release builds fine in the
+next RHEL.
+
+It's a little unfortunate we are getting this feedback too late: the upstream
```

Comment:
```suggestion
But this feedback might be a little bit too late: the upstream
```

My main concern here was that the previous sentence is using "you", this one used "we", and felt confusing. Made it impersonal to fix this.